README.md

```diff
@@ -87,11 +89,27 @@ import { AppComponent } from './app.component';
     BrowserModule
   ],
   providers: [],
-  schemas: [CUSTOM_ELEMENTS_SCHEMA],
-  bootstrap: [AppComponent]
+  schemas: [ CUSTOM_ELEMENTS_SCHEMA ],
+  bootstrap: [ AppComponent ]
 })
 export class AppModule { }
 
+// or the vanilla JS version
+
+(function(app) {
+  app.AppModule =
+    ng.core.NgModule({
+      imports: [ ng.platformBrowser.BrowserModule ],
+      declarations: [ app.AppComponent ],
+      schemas: [ ng.core.CUSTOM_ELEMENTS_SCHEMA ],
+      bootstrap: [ app.AppComponent ]
+    })
+    .Class({
+      constructor: function() {}
+    });
+})(window.app || (window.app = {}));
+
+
 /*
 You can now use the custom element in either a standalone or inline template
```

Comment:
ok I get you now - I think the readme should try to clarify that, although I understand it's complex to convey - 

that's a whole lot of boilerplate though (and I realise Angular is all about that) - but I'm still wondering whether it's possible to bypass it